app/helpers/host_helper/textual_summary.rb

```diff
@@ -395,9 +415,9 @@ def textual_users
   def textual_groups
     return nil if @record.is_vmware_esxi?
     num = @record.number_of(:groups)
-    h = {:label => "Groups", :image => "group", :value => num}
+    h = {:label => _("Groups"), :image => "group", :value => num}
     if num > 0
-      h[:title] = "Show the #{pluralize(num, 'group')} defined on this #{host_title}"
+      h[:title] = _("Show the Groups defined on this %{title}") % {:title => host_title}
```

Comment:
This should correctly be:

`h[:title] = n_("Show the Group defined on this %{title}", "Show the Groups defined on this %{title}", num) % ...`